src/main/resources/templates/job.xml

```diff
@@ -15,6 +15,14 @@
                 </parameterDefinitions>
             </hudson.model.ParametersDefinitionProperty>
         {% endif %}
+        <jenkins.model.BuildDiscarderProperty>
+            <strategy class="hudson.tasks.LogRotator">
+                <daysToKeep>{{ STORE_BUILDS_DAYS }}</daysToKeep>
+                <numToKeep>{{ STORE_BUILDS_TOTAL }}</numToKeep>
+                <artifactDaysToKeep>{{ STORE_ARTIFACTS_DAYS }}</artifactDaysToKeep>
+                <artifactNumToKeep>{{ STORE_ARTIFACTS_TOTAL }}</artifactNumToKeep>
```

Comment:
What if one or few of those fields are not set? Will they still be added in this file? I think we only probably need to check the value and if it's greater than 0, we then write it. WDYT?